// The BigQuery Storage API expects protocol buffer data to be encoded in the
// proto2 wire format. This allows it to disambiguate missing optional fields
// from default values without the need for wrapper types.

// Define a message type representing the rows in your table. The message
// cannot contain fields which are not present in the table.

// Use the required keyword for client-side validation of required fields.

syntax = "proto2";

message RedditSubmission {
	required string id = 1;
	required string title = 2;
	optional string selftext = 3;
	required string created_utc = 4;
	required string streamed_utc = 5;
}